### PR TITLE
Issue-222: Replace Raspbian references

### LINKED ISSUE #222: Documentation📄: Rasbian has become Raspberry Pi OS
### What is the current documentation state?

The OH / IEAM documentation references `Raspbian` in several pages.  It is now called `Raspberry Pi OS`.  There are two flavors of it `32bit` and `64bit`.  This GH issue should make changes to these markdown files:

```
docs/installing/adding_devices.md
docs/getting_started/faq.md
docs/getting_started/known_issues.md
```

### Where is this stated?

_No response_

### Why do you want to improve the statement?

_No response_

### Proposed Statement

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by sonaljain067**
Signed-off-by: Sonal Jain <sonaljain067@gmail.com>

# Pull Request Template

## Description
This issue fixes Raspbian in documentation to Raspberry Pi OS

Fixes #222 

## Type of change

Please delete options that are not relevant.
[✓] This change requires a documentation update

## How Has This Been Tested?
NA

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [✓] My code follows the style guidelines of this project
- [✓ ] I have made corresponding changes to the documentation
- [✓ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [✓ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

@johnwalicki Please review my Pull Request

### CODE REVIEW

**Review by joewxboy — CHANGES_REQUESTED** [review]
See inline comments

File: docs/getting_started/known_issues.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ These are known issues and limitations for {{site.data.keyword.ieam}}.
    ```
    {: codeblock}
 
-* When you are upgrading from Raspberry Pi Raspbian 9 (stretch) to Raspberry Pi Raspbian 10 (buster), a previously installed, configured, and running agent might require reinstallation. In some instances, the agent enters a restart loop that reinstallation solves. Run `ps -aux | grep "anax"` several times to observe if the agent PID is incrementing after the upgrade. This indicates whether a loop is encountered after the upgrade.
+* When you are upgrading from Raspberry Pi OS 9 (stretch) to Raspberry Pi OS 10 (buster), a previously installed, configured, and running agent might require reinstallation. In some instances, the agent enters a restart loop that reinstallation solves. Run `ps -aux | grep "anax"` several times to observe if the agent PID is incrementing after the upgrade. This indicates whether a loop is encountered after the upgrade.
```
Comment (joewxboy) [inline docs/getting_started/known_issues.md:41]: It should be:

When you are upgrading from Raspberry Pi OS 10 (buster) or Raspberry Pi OS 11 (bullseye), ...

There is no Raspberry Pi OS 9 (stretch)

File: docs/installing/adding_devices.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ The following instructions guide you through the process of installing the requi
    * {{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines that run Ubuntu 20.x (focal) or Ubuntu 18.x (bionic)
    * {{site.data.keyword.rhel}} 7.6, 7.9, 8.1, 8.2, and 8.3
 * ARM (32-bit)
-   * {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspbian buster or stretch
+   * {{site.data.keyword.linux_notm}} on ARM (32-bit), for example Raspberry Pi, running Raspberry Pi OS buster or stretch
```
Comment (joewxboy) [inline docs/installing/adding_devices.md:37]: Instead of "buster or stretch" make it "buster or bullseye"

**Review by joewxboy — CHANGES_REQUESTED** [review]

**Review by johnwalicki — APPROVED** [review]
Fantastic - Thanks

**Comment by sonaljain067** [comment]
Done with the changes!

**Review by Abhijay007 — APPROVED** [review]
LGTM

**Review by johnwalicki — APPROVED** [review]
Thanks for the clarifications.

**Review by joewxboy — APPROVED** [review]

**Review by joewxboy — APPROVED** [review]

**Pull request merged**